### DISCUSSION TITLE
Welcome to TechBuzz Discussions!

### DISCUSSION THREAD

**Discussion by sivaprasadreddy** [sivaprasadreddy/techbuzz#61] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adityachaudhari** [comment]
Hi TechBuzzers,
This is Aditya Chaudhari, day to day I help build efficient, scalable and maintainable e-commerce solutions using SAP commerce(hybris), spring framework.
I have 9+ years of experience past 7 + years working in e-commerce domain specifically on SAP Commerce ( Hybris) tool which is enterprise software based on spring framework.
I am here to learn new technologies by contributing to the repo as well as sharpen my expertise by keep contributing where i can .   This is my first open-source contribution excited to work and connect to you all.
please click here [[linkedin-profile]](https://www.linkedin.com/in/aditya-chaudhari-26379657/) to know more about me.

**Reply by sivaprasadreddy** [reply]
Hey Aditya,
Welcome to TechBuzz members.
Since TechBuzz was announced and open-sourced, you have been very active and contributing to the project. I highly appreciate it.
I hope we all learn many things while building this application.

Cheers,
Siva

**Comment by kuldeepsidhu88** [comment]
Hello,
I am Kuldeep Singh 10+ years of experience in Java, Spring eco system. 
Would be happy to contribute to TechBuzz in any way I can.

**Comment by maelstrom0x8** [comment]
hello,
i'm emmanuel godwin here to build experience with opensource projects and learn
from the community.
